apps/aecore/lib/aecore/peers/worker/supervisor.ex

```diff
@@ -1,15 +1,17 @@
 defmodule Aecore.Peers.Worker.Supervisor do
   @moduledoc """
   Supervises the Peers, PeerConnectionSupervisor and ranch acceptor
```

Comment:
"Supervises the Peers, PeerConnectionSupervisor, Sync, Jobs and ranch acceptor processes with a one_for_all strategy"